src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+interface CommentListProps {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<CommentListProps> = ({ comments }) => {
+  return comments.length
+    ? (
+      <div className="CommentList">
+        {comments.map((comment) => (
+          <CommentInfo key={comment.id} comment={comment} />
+        ))}
+      </div>
+    ) : (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    );
```

Comment:
you should always return CommentInfo from the map here and check for this condition (comments.length) inside PostInfo component